src/main/java/com/google/sps/server/FullCalendarDisplay.java

```diff
@@ -1,51 +0,0 @@
-/*
```

Comment:
Yes. This servlet was originally used to give FullCalendar the hardcoded schedules, but now since we have the Stepper in the front end, we're working on passing the input form data instead.